pkgs/sops-install-secrets/main.go

```diff
@@ -936,11 +957,15 @@ func installSecrets(args []string) error {
 				return fmt.Errorf("Cannot write key to age file: %w", err)
 			}
 		}
+		ageFile.Close()
```

Comment:
Can we move the importing the age file to its own function instead to make the `defer` happen earlier?